www/index.php

```diff
@@ -74,82 +57,167 @@
-        "/install/' is accessible. Delete this directory to prevent security problem.";
-    $file_install_access = 1;
-}
-
-/**
- * Install frontend assets if needed
- */
-$requestUri = filter_var(
-    $_SERVER['REQUEST_URI'],
-    FILTER_SANITIZE_STRING,
-    [
-        'options' => [
-            'default' => '/centreon/'
-        ]
-    ]
-);
-$basePath = '/' . trim(explode('index.php', $requestUri)[0], "/") . '/';
-$basePath = str_replace('//', '/', $basePath);
-$indexHtmlPath = './index.html';
-$indexHtmlContent = file_get_contents($indexHtmlPath);
-
-// update base path only if it has changed
-if (!preg_match('/.*<base\shref="' . preg_quote($basePath, '/') . '">/', $indexHtmlContent)) {
-    $indexHtmlContent = preg_replace(
-        '/(^.*<base\shref=")\S+(">.*$)/s',
-        '${1}' . $basePath . '${2}',
-        $indexHtmlContent
-    );
-
-    file_put_contents($indexHtmlPath, $indexHtmlContent);
-}
-
 CentreonSession::start();
 
-if (isset($_GET["disconnect"])) {
-    $centreon = &$_SESSION["centreon"];
-
-    /*
-     * Init log class
-     */
-    if (is_object($centreon)) {
-        $CentreonLog = new CentreonUserLog($centreon->user->get_id(), $pearDB);
-        $CentreonLog->insertLog(1, "Contact '" . $centreon->user->get_alias() . "' logout");
-
-        $pearDB->query("DELETE FROM session WHERE session_id = '" . session_id() . "'");
-
-        $sessionStatement = $pearDB->prepare("DELETE FROM security_token WHERE token = :sessionId");
-        $sessionStatement->bindValue(':sessionId', session_id(), \PDO::PARAM_STR);
-        $sessionStatement->execute();
-
-        CentreonSession::restart();
-    }
-}
-
 /*
  * Already connected
  */
 if (isset($_SESSION["centreon"])) {
     $centreon = &$_SESSION["centreon"];
-    header('Location: main.php');
+    header('Location: index.html');
 }
 
 /*
  * Check PHP version
  *
- *  Centreon >= 18.10 doesn't support PHP < 7.1
+ *  Centreon >= 22.04 doesn't support PHP < 8.0
  *
  */
-if (version_compare(phpversion(), '7.1') < 0) {
-    echo "<div class='msg'> PHP version is < 7.1. Please Upgrade PHP</div>";
+if (version_compare(phpversion(), '8.0') < 0) {
+    echo "<div class='msg'> PHP version is < 8.0. Please Upgrade PHP</div>";
 } else {
-    include_once "./include/core/login/login.php";
+    if (
+        isset($_POST["centreon_token"])
+        || (
+            isset($_GET["autologin"]) && $_GET["autologin"]
+            && isset($generalOptions["enable_autologin"])
+            && $generalOptions["enable_autologin"])
+        || (
+            isset($_POST["autologin"]) && $_POST["autologin"]
+            && isset($generalOptions["enable_autologin"])
+            && $generalOptions["enable_autologin"])
+        || (
+            !isset($generalOptions['sso_enable'])
+            || $generalOptions['sso_enable'] == 1)
+        || (
+            !isset($generalOptions['openid_connect_enable']) || $generalOptions['openid_connect_enable'] == 1)
```

Comment:
just to be consistent with previous condition
```suggestion
            !isset($generalOptions['openid_connect_enable'])
            || $generalOptions['openid_connect_enable'] == 1)
```